etc/kayobe/environments/ci-aio/automated-setup.sh

```diff
@@ -24,7 +24,7 @@ if type dnf; then
     sudo dnf -y install git
 else
     sudo apt update
-    sudo apt -y install gcc git libffi-dev python3-dev python-is-python3
+    sudo apt -y install gcc git libffi-dev python3-dev python-is-python3 python3-venv
```

Comment:
I think we need it in yoga.

Reply:
and should we update the docs to match?

Reply:
Will do. Did you come across anything else during the deployment you want to add?